src/util/indexrange.h

```diff
@@ -67,58 +67,58 @@ class IndexRange final: private std::pair<SINT, SINT> {
     }
 
     SINT length() const {
-        return (head() <= tail()) ? (tail() - head()) : (head() - tail());
+        return (start() <= end()) ? (end() - start()) : (start() - end());
     }
 
-    // Clamps index by this range including both head() and tail()
+    // Clamps index by this range including both start() and end()
     // boundaries.
     SINT clamp(SINT index) const {
```

Comment:
it is not clear that the index ins clamped-
clampIndex() .. or .. getNearestInBounds()